### ISSUE TITLE
Product admin panel 'custom options' with 2 store view.

### ISSUE THREAD

**Issue by krzysztofkacela** [magento/magento2#6640]
### Preconditions
1. Magento CE 2.0.7
2. Magento CE 2.0.9
3. Magento CE 2.1.1
4. PHP Version 5.6.25
### Steps to reproduce
1. Stores > All Stores > Create Store View
2. Add 2 store views
3. Product > Catalog > Add Product.
4. Click Advence settings > Custom Options > Add options.
   https://gyazo.com/46cc685096b66b320c3437d6c5c7b154
5. Change store view to other. Now custom options look like this.
   https://gyazo.com/a5fc5cb5551530562c0395121e857109

After upgrade to Magento CE 2.0.9 problem still exist.

After install fresh Magento CE 2.1.1 problem looks to be fixed BUT now when I change store view and change anything, it's saving content to "All Store View" not for chosen view.
### Expected result
1. Not showing code but checkbox to disable "use default"
   or in CE 2.1.1 saving options for store view not all store
### Actual result
1. Already described upper.

**Comment by krzysztofkacela** [comment]
Hey, any fix for this issue?

**Comment by mileswilhelm** [comment]
Hey there,

I ran into this issue this week. We are running Magento 2.0.10.

Looking into the files where the HTML is created for the checkboxes, a fellow developer and I stumbled upon the solution (and it's silly).

In `/vendor/magento/module-catalog/view/adminhtml/templates/catalog/product/edit/options/option.phtml` 
and `/vendor/magento/module-catalog/view/adminhtml/templates/catalog/product/edit/options/type/select.phtml` the templates are created and pull the HTML for the checkbox with relevant JSON data from this file: `/vendor/magento/module-catalog/Block/Adminhtml/Product/Edit/Tab/Options/Option.php`.

These phtml files use Underscore.js to pull the JSON data that go into the attributes for the checkbox to work. It also is responsible for pulling the checkbox HTML created in `/vendor/magento/module-catalog/Block/Adminhtml/Product/Edit/Tab/Options/Option.php`. [http://i.imgur.com/h09PbtE.png](http://i.imgur.com/h09PbtE.png)

We messed with the weird tags pulling the data to try to find why it was only displaying as a string rather than HTML. This led us to an error in the Underscore.js file. One google search later and we learned the weird tags <%- %> come from Underscore.js. Relevant info from the Underscore.js documentation: [http://i.imgur.com/ZOzxFN9.png](http://i.imgur.com/ZOzxFN9.png)

**So, when they use the <%- tag prior to the data variable responsible for displaying the HTML for the checkboxes, it escapes HTML code. Replace the <%- tags with <%= tags to make it display as it should be, and the problem is solved.**

# tl;dr
It's a coding error where the wrong tag from Underscore.js was used. 

In `/vendor/magento/module-catalog/view/adminhtml/templates/catalog/product/edit/options/option.phtml` - line 52 and `/vendor/magento/module-catalog/view/adminhtml/templates/catalog/product/edit/options/type/select.phtml` - line 47, change **<%-** data.checkboxScopeTitle %> to **<%=** data.checkboxScopeTitle %>.

**Comment by shaibar** [comment]
CE 2.1.2
Still happens
Create a product with 2 store view
Add custom options
Change custom option title / values for radio button / checkbox
Custom option title and values are changing in All Store Views and not in the specific store view

**Comment by keinereiner** [comment]
still in 2.1.7 exists this problem....

**Comment by oleg-onufer** [comment]
Hi @krzysztofkacela.
 This issue is a duplicate  for this issue  https://github.com/magento/magento2/issues/7954

**Comment by veloraven** [comment]
I'm closing this issue as duplicate of #7954
